miden-crypto/src/word.rs

```diff
@@ -3,13 +3,18 @@ use core::{
     cmp::Ordering,
     fmt::Display,
     hash::{Hash, Hasher},
-    ops::Deref,
+    ops::{Deref, Range},
     slice,
 };
 
 use thiserror::Error;
+use winter_crypto::Digest;
 
-use super::{DIGEST_BYTES, DIGEST_SIZE, Digest, Felt, StarkField, ZERO};
+const WORD_RANGE: Range<usize> = 4..8;
+const WORD_SIZE: usize = WORD_RANGE.end - WORD_RANGE.start;
+const WORD_BYTES: usize = 32;
```

Comment:
```suggestion
const WORD_SIZE_FELT: usize = WORD_RANGE.end - WORD_RANGE.start;
const WORD_SIZE_BYTES: usize = 32;
```
This is clearer to me; WDYT?